### PR TITLE
UG: Add FAQ, quick start, fix bug of writing unarchive active client, add purpose of archive

### LINKED ISSUE #142: Add FAQ into UG

### LINKED ISSUE #143: Add quick start into UG

### LINKED ISSUE #163: [PE-D] Add in purpose of archive into UG
It is not specified what is the purpose of the archived list.
If it is indeed to keep a record of the contacts, then I believe deleting a policy shouldn't affect this list.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: erinmayg/ped#6

### LINKED ISSUE #168: [PE-D] In UG, should be unarchiving an archived client instead of active client
Was it meant to be 'Unarchiving an archived client'?

-------------
Labels: `severity.High` `type.DocumentationBug`
original: onglijin/ped#2

### PR DESCRIPTION
**Pull request by WeiJie96**
Close #142 
Close #143 
Close #168 
Close #163

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -12,9 +12,36 @@ Interface (CLI) while still having the benefits of a Graphic User Interface (GUI
...
+## Quick start
+
+1. Ensure you have Java `11` installed in your Computer. 
+   You can check the version by opening a command window, and running the `java -version` command.
+
+2. Download the latest `[CS2103-T16-2][Insurance4Insurance].jar` from [here](https://github.com/AY2021S1-CS2103-T16-2/tp/releases).
+
+3. Copy the file to the **empty folder** you want to use as the _home folder_.
+
+4. Launch the app by using the `java -jar` command (do not use double-clicking).
```
Comment (Joven-Heng) [inline docs/UserGuide.md:24]: Same thing. The double clicking thing. If AB3 told us to launch using -jar then yeah sure

**Review by Joven-Heng — APPROVED** [review]
LGTM if the first part is ok. I think the rest is fine

File: docs/UserGuide.md (hunk 1 above)
Comment (WeiJie96) [reply docs/UserGuide.md:24]: Same for here, I followed the PE instructions

**Comment by Joven-Heng** [comment]
LGTM

**Pull request merged**